### DISCUSSION TITLE
When i set "dpgen run param.json machine.json ",occured this error.

### DISCUSSION THREAD

**Discussion by qinmi-sudo** [deepmodeling/dpgen#480] [Q&A]
![image](https://user-images.githubusercontent.com/70270603/126887676-9c897523-0ffc-4551-a7f0-71a985c2dd1b.png)
![image](https://user-images.githubusercontent.com/70270603/126887694-308bc42a-843e-49b8-ad57-44c6e67548c9.png)

**Comment by njzjz** [comment]
Please check `sys_configs_prefix` and `sys_configs`. Did you set it to `/`?

**Reply by qinmi-sudo** [reply]
"init_data_prefix" : "/lustre/ISSP2/bqcao/mqin/Dpgen/test/Initial",
    "init_data_sys":    [
                "initset/deepmd"
        ],
   "sys_configs_prefix": "/lustre/ISSP2/bqcao/mqin/Dpgen/test/Exploration",
    "sys_configs":      [
    "configs/POSCAR"
        ],

**Comment by njzjz** [comment]
here
```json
"sys_configs": [
"configs/POSCAR"
],
```

The correct one is
```json
"sys_configs": [
[
"configs/POSCAR"
]
],
```
